api/v1/types/util.go

```diff
@@ -63,3 +69,23 @@ func (p *GetLayerStatsActiveAccountsParams) Validate() error {
 	}
 	return &InvalidParamFormatError{ParamName: "window_step_seconds", Err: fmt.Errorf("invalid value: %d", *p.WindowStepSeconds)}
 }
+
+// The autogenerated server performs type validation that the input string
+// conforms to either an oasis bech32-encoded address or an ethereum
+// address, but we double check just to be sure.
+func UnmarshalToOcAddress(raw *string) (*StakingAddress, error) {
```

Comment:
Nit: when speaking of addresses, `raw` to me implies unencoded bytes. The type on this line is clear, but it makes for stilted reading later on. Consider renaming to `text` (my favorite, for parallels with `UnmarshalText()`) or `encoded` or `serialized` or similar.

Reply:
Good idea, updated!